src/EnergyPlus/EvaporativeCoolers.cc

```diff
@@ -3602,6 +3604,11 @@ void GetInputZoneEvaporativeCoolerUnit(EnergyPlusData &state)
                 }
             }
 
+            if (!lNumericBlanks(4)) {
+                // Shut Off Relative Humidity
+                thisZoneEvapUnit.ShutOffRelativeHumidity = Numbers(4);
```

Comment:
OK, so if it is blank, it stays as the default 100.  This seems reasonable ... though I guess there is a possibility that some user file is currently still dumping moisture into the space even when the RH is at 100, and now with this change we are quietly turning it off.  The other option is to keep a bool to decide if this field was blank, and if so, use the exact same behavior as before.  I'm not suggesting we do that, just walking myself through the changes.  Leave it, this is fine.

Reply:
okay cool